### ISSUE TITLE
So for Google Pixel phones running Android 14

### ISSUE THREAD

**Issue by biggerald1** [shmykelsa/AAAD#267]
Any hope for phones on 14? Not 1 app in AAAD works on 14 now on my phone unfortunately.

             So for Google Pixel phones running Android 13:

We have identified an issue on Android 13 for Pixels.

Can you please test downloading this APK (https://www.apkmirror.com/apk/google-inc/package-installer/package-installer-13-release/package-installer-13-android-apk-download/) and then try uninstalling all the apps and installing them again.

_Originally posted by @shmykelsa in https://github.com/shmykelsa/AAAD/issues/247#issuecomment-1634200543_

**Comment by muppetofdoom** [comment]
Hi, i've tried on pixel 6 pro android 14 and it won't let me instal;l the apk, it just says you can't install the app on your device :( 
here's hoping a fix is coming soon :)

**Comment by pictureknight** [comment]
Pixel 7a with Android 14. I can confirm it wouldn't let me downgrade to packageinstaller 13, even using adb with -d tag.

**Comment by justinglavin** [comment]
I downgraded my Pixel 6 from Android 14 to Android 13 and CarStream is working great. I will probably attempt updating to 14 eventually since it seems like if the package installer fix is installed on 13 first it should work on 14.

**Comment by biggerald1** [comment]
It should as it has for ppl in ur same situation. They have updated a14 twice though w/ patches so let us know

Sent via Outlook
________________________________

**Comment by christopherquenneville** [comment]
Android 14 not working - has anyway found an APK like the one that was working in Android 13?

**Comment by justinglavin** [comment]
You need to downgrade to 13

**Comment by biggerald1** [comment]
No, I wish. I wish the developer would give us any type of update like is it being worked on or is it dead? If it's dead can u refund us tht got it recently? Can u quit SELLING it or at least put a giant warning tht it doesn't work on a14? 

U can downgrade MOST phones to a13 using a computer & an online guide IF ur phones unlocked. If it's locked ur stuck, do NOT try. Also, you'll lose ur files so upload them to a good cloud storage or safe em on the computer

**Comment by yeahforbes** [comment]
Got an update from the developer. There seem to be about 4 duplicate issues on this, so I'll just drop a link: https://github.com/shmykelsa/AAAD/issues/269#issuecomment-1821844148
